cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -696,6 +721,22 @@ private V1PodSpec createPodSpec(Location runtimeConfigLocation,
     volumeMounts.add(new V1VolumeMount().name("workdir").mountPath(workDir));
     volumeMounts.addAll(Arrays.asList(extraMounts));
 
+    // Add external secrets as secret volume mounts
+    List<V1Volume> secretVolumes = new ArrayList<>();
+    if (sConfMountedRunnables.contains(runnableName)) {
+      String mountPath = masterEnvContext.getConfigurations().get(POD_SCONF_MOUNT_PATH);
```

Comment:
Using cConf to enable/disable mounting secrete, this means either all twill applications created by appfab will have secrete either always mounted or never mounted. Cannot be configured on per twill application basis, right?

Reply:
So are we suppose to have the CDAP operator to setup this configuration since the secret name and mount path is determined by the operator?

Reply:
Not necessarily; all runnables which are prepared using a single preparer will have the sConf mounted, and that is determined by calls to the withSecuritySecret(runnableName) function.